packages/playwright-core/bundles/utils/package.json

```diff
@@ -13,6 +13,7 @@
     "commander": "8.3.0",
     "debug": "^4.3.4",
     "glob": "^7.1.3",
+    "http-proxy": "^1.18.1",
```

Comment:
Can we only install this in the container? Playwright does not use it.

Reply:
As discussed offline with @dgozman, there are two options: either have all of container story inside of playwright-core, or extract it into a separate package, e.g. @playwright/container

Originally, we have docker implemented in playwright-test, but moved it to the playwright-core so that language ports would be able to benefit from it.

So having this as part of playwright-core is an intentionatl design decision.